Categories Defect

elviejo79 opened the pull request:
there was a defect in sites.go
it was only indexing one category and one tag per post.
it was pretty easy to spot
ok == true

Commit:
There was a bug in categories and tags, and it only indexed one element in the map

Commit:
refs ljzac/ljzac.github.io#6 en la compilacion anterior teniamos de nuevo el error de una sola nota por category... este error estaba para minutos catastroficos

bradrydzewski commented:
thanks @elviejo79 for the bug fix, and for formatting the code! quick question -- how does MainImg() work?

Commit:
since goyaml didn't support unicode in the structure I had to use goyaml2

Commit:
Para dismunir el numero de categorias

Commit:
Agregando hora a la función date_to_string

En la función date_to_string que solo regresaba la fecha se modificó para que regrese también la hora.

Commit:
refs #7 si tenemos de 600 posts usa todas las disponibles

AAlvz committed:
REFS #7. Mejorando la imagen principal

Commit:
refs #30, refs #7. Para que ponga un texto introductorio mas largo en la nota

Commit:
refs #32 tienes la propiedad .site.MinuteByMinute

Commit:
corrigiendo errores del colega pues ya que

Commit:
modified authors

Commit:
por si me faltó subir algo

Commit:
refs #39 implementa el img de diferentes sizes

Commit:
agregué dos funciones, una para cortar el mapa al entero dado y otra para tomar las noticias de los autores en específico

Commit:
refs #60 hacer que en el minuto por minuto sólo se vean ciertas categorías, se cambió el site.go del jkl

Commit:
arreglar el problema de los links relativos para que sean absolutos refs #3

AAlvz committed:
Mejoras a metodo de imagenes

Commit:
refs ljzac/ljzac.github.io#9 solo mostrará los posts publicados por el editor

Commit:
refs ljzac/ljzac.github.io#58 función para obtener video de youtube en portada

Commit:
refs ljzac/ljzac.github.io#68 agregue el video thumbnail ademas del metodo ExtractFragment

Commit:
refs #44 para que puedan cambiar una seccion, y permitir templates anidados hice un render recursivo de los templates

Commit:
refs #44 elimin'e prints innecesarios

Commit:
refs ljzac/ljzac.github.io#50 modifique el procesamiento de los posts, para que la fecha se tome de los directorios, no del nombre del archivo

Commit:
refs ljzac/ljzac.github.io#50 para que ponga el nombre del archivo sin considerar la fecha

Commit:
refs #79 primero asegurarnos de que el binary jkl/jkl/ se pueda subir

Commit:
refs #79 primero asegurarnos de que el binary jkl/jkl/ se pueda subir

Commit:
refs #56 ya considera las categorias de otras noticias

Commit:
refs #75 errores de compilacion

Commit:
refs ljzac/ljzac.github.io#100 para que acepte a category

Commit:
quitamos las publicidades de las noticias día a día

Commit:
bajo el número de elementos DOM al poner sólo 60 noticias en la sección noticias al día

Commit:
limita el numero de noticias

Commit:
para cortar el numero de noticias

Commit:
el nuevo binary

Commit:
refs ljzac/ljzac.github.io#143 Quite la hora de los posts

Commit:
refs ljzac/ljzac.github.io#163 para que minute by minute tambien se pueda accesar como una categoria

Commit:
merge

Commit:
new compilation

Commit:
cambie el intro text para que regrese 30 palabras completas en lugar de caracteres

Commit:
refs ljzac/ljzac.github.io#175 agregando posts para la sección Zacatecas

elviejo79 committed:
agrgue la función para desplegar la fecha en el rss.xm

iLeonelPerea committed:
Cambio en la funcion GetIntrotext

En lugar de cerrar y abrir p se puso un salto de línea

iLeonelPerea committed:
test

iLeonelPerea committed:
Revert "test"

This reverts commit c33e030331f3d6aa974a41091386a3b17275e107.

iLeonelPerea committed:
test..

iLeonelPerea committed:
Revert "test.."

This reverts commit 065941567393ef917291ecbeff7d304650cd9021.

iLeonelPerea committed:
test

iLeonelPerea committed:
Revert "test"

This reverts commit 4601ee95cac1a9ad742ba483e09db433f2e28324.

iLeonelPerea committed:
todos
